.github/workflows/pending-set.yml

```diff
@@ -0,0 +1,29 @@
+name: "set pending status"
+
+on:
+  pull_request_target:
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+    - name: set pending status
+      if: github.repository_owner == 'NixOS'
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GSU_VERSION: "0.5.0"
+        GSU_URL: "https://github.com/cloudposse/github-status-updater/releases/download"
+      run: |
+        curl -sSf -O -L -C - \
+        "$GSU_URL/$GSU_VERSION/github-status-updater_linux_amd64" && \
+        chmod +x github-status-updater_linux_amd64 && \
+        ./github-status-updater_linux_amd64 \
+          -action update_state \
+          -token "$GITHUB_TOKEN" \
+          -owner NixOS \
+          -repo nixpkgs \
+          -state failure \
+          -context "Wait for ofborg" \
+          -description "This failed status will be cleared when ofborg finishes eval." \
```

Comment:
For now I've changed this from pending to failure, currently ofborg only sends `check_suite` at the end of eval so a PR that fails early will stay as pending which might be more confusing for contributors.

https://github.com/NixOS/ofborg/pull/527 should allow us to use pending.

Reply:
Does this failure state triggers an email?

Reply:
I don't think so?